src/dialects/abstract/query-interface.js

```diff
@@ -421,13 +426,13 @@ export class QueryInterface {
        * it will not throw an error like built-ins do (e.g. DESCRIBE on MySql).
        */
       if (_.isEmpty(data)) {
-        throw new Error(`No description found for "${tableName}" table. Check the table name and schema; remember, they _are_ case sensitive.`);
+        throw new Error(`No description found for "${table.tableName}" table. Check the table name and schema; remember, they _are_ case sensitive.`);
```

Comment:
we could include the schema here (and in the next error message)

Reply:
Should be done, just need to fix the integration test